docs/dev-setup.md

```diff
@@ -54,6 +54,15 @@ Then start Tilt:
 make tilt-up
 ```
 
+The Tilt dashboard can be accessed at http://127.0.0.1:10350. If tilt is running remotely, it can be accessed using ssh tunneling. An example of ```~/.ssh/config```  is show below.
```

Comment:
```suggestion
The Tilt dashboard can be accessed at <http://127.0.0.1:10350>. If tilt is running remotely, it can be accessed using ssh tunneling. An example of ```~/.ssh/config```  is show below.
```

Reply:
Thanks, fixed.